docs/docs/themes.md

```diff
@@ -16,6 +16,57 @@ Once you're ready to swap to a theme, follow the steps described in [🚀Install
 
 Themes with `minimal` in their names do not require a Nerd Font. Read about [🆎Fonts][fonts] for more information.
 
+To set a random theme everytime you open terminal, you can use the following:
+
+###### Windows PowerShell
```

Comment:
please rename the file to `.mdx` and add tabs (see any other `.mdx`) file for reference.